src/css/property-descriptors/box-shadow.ts

```diff
@@ -24,8 +24,7 @@ export const boxShadow: IPropertyListDescriptor<BoxShadow> = {
         if (tokens.length === 1 && isIdentWithValue(tokens[0], 'none')) {
             return [];
         }
-
-        return parseFunctionArgs(tokens).map((values: CSSValue[]) => {
+        let c = parseFunctionArgs(tokens).map((values: CSSValue[]) => {
```

Comment:
This change introduces no improvement and is actually more confusing due to unnamed variables.